app/models/podcast_import.rb

```diff
@@ -6,7 +6,7 @@ class PodcastImport < ApplicationRecord
 
   serialize :config, HashSerializer
 
-  attr_accessor :feed, :feed_raw_doc, :templates, :feed_source
+  attr_accessor :feed, :feed_raw_doc, :templates, :feed_source, :import_metadata
```

Comment:
Curious - why do you need the attr_accessor here?  I don't see it being accessed anywhere.

Reply:
maybe a habit from a different ticket. for some reason i thought it was necessary, but i suppose not!